cmd/buck/cli/archive.go

```diff
@@ -68,18 +69,20 @@ var bucketArchiveStatusCmd = &cobra.Command{
 			fmt.Printf("\n")
 			cmd.Message("Cid logs:")
 			ch := make(chan string)
-			wCtx, cancel := context.WithCancel(ctx)
+			wCtx, cancel := context.WithCancel(context.Background())
 			defer cancel()
 			go func() {
 				err = clients.Buckets.ArchiveWatch(wCtx, key, ch)
 				close(ch)
 			}()
 			for msg := range ch {
 				cmd.Message("\t %s", msg)
-				r, err := clients.Buckets.ArchiveStatus(ctx, key)
+				sctx, scancel := context.WithTimeout(context.Background(), time.Second*3)
```

Comment:
The duration could be stored with the other command time out variable.

Reply:
Done. I'll wait for CI and merge. Thanks!